src/main/java/org/littil/api/module/service/Module.java

```diff
@@ -0,0 +1,19 @@
+package org.littil.api.module.service;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.validation.constraints.NotEmpty;
+import java.util.UUID;
+
+@Data
+public class Module {
+    private UUID id;
+
+    @NotEmpty(message = "{Module.name.required}")
+    @Column(name = "module_name")
```

Comment:
I don't think we need @Column() in the DTO?

Reply:
removed @Column annotation